common.blocks/menu-item/menu-item.spec.js

```diff
@@ -37,6 +39,26 @@ describe('menu-item', function() {
         });
     });
 
+    describe('disabled', function() {
+        it('should set "aria-disabled" attribute if disabled', function() {
+            menuItem.setMod('disabled', true);
+            expect(menuItem.domElem.attr('aria-disabled')).to.equal('true');
```

Comment:
BTW, it's a good practice to check that particular attribute hadn't been set, before first `setMod` call.